src/app-layout/__integ__/runtime-drawers.test.ts

```diff
@@ -68,3 +68,96 @@ for (const visualRefresh of [true, false]) {
     );
   });
 }
+
+describe('Visual refresh toolbar only', () => {
+  function setupTest(testFn: (page: BasePageObject) => Promise<void>) {
+    return useBrowser(async browser => {
+      const page = new BasePageObject(browser);
+
+      await browser.url(
+        `#/light/app-layout/runtime-drawers?${new URLSearchParams({
+          hasDrawers: 'false',
+          hasTools: 'true',
+          splitPanelPosition: 'side',
+          visualRefresh: 'true',
+          appLayoutToolbar: 'true',
+        }).toString()}`
+      );
+      await page.waitForVisible(wrapper.findDrawerTriggerById('security').toSelector(), true);
+      await testFn(page);
+    });
+  }
+
+  test(
+    'displays only the most recently opened drawer in a full-width popup on mobile view (global drawer on top of the local one)',
+    setupTest(async page => {
+      await page.click(wrapper.findDrawerTriggerById('security').toSelector());
+      await page.click(wrapper.findDrawerTriggerById('circle-global').toSelector());
+
+      await page.setWindowSize(viewports.mobile);
+      // technically, both drawers are present in the DOM tree, but only one is visible.
+      // the isClickable check ensures that the drawer is actually visible
+      await expect(page.isClickable(wrapper.findDrawerById('circle-global').toSelector())).resolves.toBe(true);
```

Comment:
Because this test checks the behavior on mobile view, when multiple drawers are present in the DOM, but only one is shown in the page. But `page.isDisplayed` checks if the element is displayed _even when the element is outside the viewport_, so in this case we need to check if the element is actually shown in the page and a user can interact with it.

Reply:
How is it different from [the current tools behavior](https://github.com/cloudscape-design/components/blob/main/src/app-layout/visual-refresh-toolbar/drawer/index.tsx#L111) and why does it have to be different?

Reply:
The current behavior does not have such a case because only one drawer could be opened. This test covers the following requirement:

> If more than one drawer is open (including a local drawer) and the screen is resized to mobile view, only one drawer should be shown in fullscreen mode, following this prioritization: 
1. The currently focused active drawer. 
2. If no drawer is focused, the last opened active drawer.

Reply:
desktop screen: multiple drawers are shown
resize to mobile screen
mobile screen: only one drawer is shown and other ones are hidden because their z-indexes are less than on the active one

Reply:
no, it does not

Reply:
Summary from our meeting sync. When drawers do not fit on the screen, they should be closed, either hidden if the "preserve" property is set or unmounted if not.

Reply:
addressed